Sources/RemoteMessaging/RemoteMessagingStore.swift

```diff
@@ -60,61 +73,77 @@ public final class RemoteMessagingStore: RemoteMessagingStoring {
         self.notificationCenter = notificationCenter
         self.errorEvents = errorEvents
         self.remoteMessagingAvailabilityProvider = remoteMessagingAvailabilityProvider
+        self.context = database.makeContext(concurrencyType: .privateQueueConcurrencyType, name: Constants.privateContextName)
 
         featureFlagDisabledCancellable = remoteMessagingAvailabilityProvider.isRemoteMessagingAvailablePublisher
-            .map { !$0 }
+            .filter { !$0 }
             .removeDuplicates()
             .sink { [weak self] _ in
-                self?.deleteScheduledMessagesIfNeeded()
```

Comment:
it has been renamed to deleteScheduledMessages, and the check for "ifNeeded" is performed before calling the delete function.